tests/unit_tests/test_images.py

```diff
@@ -1,10 +1,15 @@
 import pytest
 
 import numpy as np
-from pynwb.image import GrayscaleImage
+from pynwb.testing.mock.base import mock_TimeSeries
```

Comment:
As much as I like the idea of using mocks throughout our tests, we'll have to give it a while longer until our minimal testing version is the version of PyNWB that introduced them